### PR TITLE
Add media timeout check to MediaConn

### PR DESCRIPTION
**Pull request by Sean-Der**
If no RTP packets are seen for 30 seconds timeout the call

### CODE REVIEW

File: pkg/media/rtp/conn.go [hunk 1]
```diff
@@ -145,3 +155,25 @@ func (c *Conn) ReadRTP() (*rtp.Packet, *net.UDPAddr, error) {
...
 }
+
+func (c *Conn) onTimeout(timeoutCallback func()) {
+	go func() {
+		ticker := time.NewTicker(timeoutCheckInterval)
+		defer ticker.Stop()
+
+		var lastPacketCount uint64
+		for range ticker.C {
+			if c.closed.IsBroken() {
```
Comment (dennwc) [inline pkg/media/rtp/conn.go]: You could wait on both `ticker.C` and `c.closed.Watch()` in a select.

File: pkg/media/rtp/conn.go [hunk 2]
```diff
@@ -18,24 +18,32 @@ import (
...
-	return &Conn{
+const (
+	timeoutCheckInterval = time.Second * 30
+)
+
+func NewConn(timeoutCallback func()) *Conn {
+	c := &Conn{
 		readBuf: make([]byte, 1500), // MTU
 	}
+	c.onTimeout(timeoutCallback)
```
Comment (dennwc) [inline pkg/media/rtp/conn.go]:
```suggestion
	if timeoutCallback != nil {
		c.onTimeout(timeoutCallback)
	}
```

File: pkg/sip/config.go [hunk 3]
```diff
@@ -99,48 +97,3 @@ func getLocalIP(localNet string) (string, error) {
 
 	return "", fmt.Errorf("No local interface found")
 }
-
-var listenErr = fmt.Errorf("Failed to listen on UDP Port")
-
-func listenUDPInPortRange(portMin, portMax int, IP net.IP) (*net.UDPConn, error) {
```
Comment (Sean-Der) [inline pkg/sip/config.go]: Unused function, not related PR

**Review by dennwc — APPROVED** [review]
Only a few nits, LGTM!

**Pull request merged**